src/Propel/Generator/Reverse/SqliteSchemaParser.php

```diff
@@ -150,6 +150,7 @@ protected function parseTables(Database $database, ?Table $filterTable = null):
 
         $sql = str_replace('%filter%', $filter, $sql);
 
+        /** @var array $dataFetcher */
         $dataFetcher = $this->dbh->query($sql);
```

Comment:
Return type is actually `\Propel\Runtime\DataFetcher\DataFetcherInterface|\PDOStatement|false`. I think we can we do:
```php
        /** @var \Traversable $dataFetcher */
        $dataFetcher = $this->dbh->query($sql);
```